app/Bus/Handlers/Events/Incident/SendIncidentEmailNotificationHandler.php

```diff
@@ -56,6 +57,18 @@ public function __construct(MailQueue $mailer, Subscriber $subscriber)
      */
     public function handle(IncidentWasReportedEvent $event)
     {
+        // Notify directly specified e-mail addresses
+        $directNotifications = explode(',', $event->incident->directNotify);
+        $directNotifications = array_map('trim', $directNotifications);
+        foreach ($directNotifications as $email) {
+            if (filter_var($email, FILTER_VALIDATE_EMAIL)) {
+                $addressee = new stdClass();
```

Comment:
an alternative being instantiating a new Subscriber, or making it an array and casting it to `(object)`.